.github/workflows/lint.yaml

```diff
@@ -44,5 +44,10 @@ jobs:
             REGISTRY=$(grep "REGISTRY_URL := " $MAKEFILE | cut -d\  -f3)
```

Comment:
FYI: @amandakarina  This file is automatically generated/reconciled (`.github/workflows/lint.yaml`).  Rather than directly editing this file, any changes to the ENVs should be made here: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf#L846